final-workflow.yml

```diff
@@ -76,16 +76,18 @@ deepPATH_bigwig: [ #the containing directory for bigwig files generated from the
 scaffold/bigwig
 ]
 #-------------------------------------------------------------------------------
-host: [ #host where Apollo2 lives
+#where apollo2 lives
```

Comment:
We'll actually need 2 sets of inputs for Apollo - 1 set for the staging environment, the 2nd set for the production environment. apolloServer-workflow.cwl should use the production environment variables

Reply:
Yes! I can modify it.